lib/cdo/rake_utils.rb

```diff
@@ -183,10 +183,10 @@ def self.npm_update_g(*args)
   end
 
   def self.npm_install(*args)
+    sudo = CDO.npm_use_sudo ? 'sudo' : ''
     commands = []
     commands << 'PKG_CONFIG_PATH=/usr/X11/lib/pkgconfig' if OS.mac?
-    commands << 'sudo' if CDO.npm_use_sudo
-    commands += 'npm prune && npm update --quiet'.split(' ')
+    commands += "#{sudo} npm prune && #{sudo} npm update --quiet".split(' ')
```

Comment:
Note that space `' '` is the default argument for Ruby's `String.split` so could just `.split`.